datalad/support/annexrepo.py

```diff
@@ -0,0 +1,67 @@
+"""
+Interface to git-annex by Joey Hess.
+See https://git-annex.branchable.com/.
+"""
+__author__ = 'Benjamin Poldrack'
+
+import os
+from os.path import join, exists
+
+from gitrepo import GitRepo
+import datalad.log
+
+from datalad.cmd import Runner
+
+
+class AnnexRepo(GitRepo):
+    """
+    Representation of an git-annex repository.
+
+    """
+
+    def __init__(self, path, url=None):
+        """
+        AnnexRepo is initialized by giving a path to the annex.
+        If no annex exists at that location, a new one is created.
+        Optionally give url to clone from.
+
+        :param path: path to git-annex repository
+        :param url: valid git url. See http://www.kernel.org/pub/software/scm/git/docs/git-clone.html#URLS
```

Comment:
Ok. Wasn't aware. Will fix it.

2015-02-23 16:59 GMT+01:00 Yaroslav Halchenko <notifications@github.com>:

> In datalad/support/annexrepo.py
> <https://github.com/datalad/datalad/pull/42#discussion_r25174905>:
>
> > +
> > +
> > +class AnnexRepo(GitRepo):
> > +    """
> > +    Representation of an git-annex repository.
> > +
> > +    """
> > +
> > +    def __init__(self, path, url=None):
> > +        """
> > +        AnnexRepo is initialized by giving a path to the annex.
> > +        If no annex exists at that location, a new one is created.
> > +        Optionally give url to clone from.
> > +
> > +        :param path: path to git-annex repository
> > +        :param url: valid git url. See http://www.kernel.org/pub/software/scm/git/docs/git-clone.html#URLS
>
> we are using numpy-styled docstrings parameters/returns description. See
> https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#id6
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/datalad/datalad/pull/42/files#r25174905>.
>